tests/ui/half-open-range-patterns/half-open-range-pats-inclusive-match-arrow.stderr

```diff
@@ -15,5 +15,5 @@ error: expected one of `=>`, `if`, or `|`, found `>`
 LL |         74..=> {},
    |              ^ expected one of `=>`, `if`, or `|`
```

Comment:
You could also probably suppress this error (delaying it as a bug, perhaps) by checking if the previous token is `..=` in the match arm parsing code?